server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/RecipientRewriteTableMock.java

```diff
@@ -18,12 +18,18 @@
  ****************************************************************/
 package org.apache.james.transport.mailets;
```

Comment:
Is it possible to replace this class with **mockito** ? It is only used for two tests in org.apache.james.transport.mailets.RecipientRewriteTableTest